src/mpid/ch4/netmod/ofi/ofi_win.h

```diff
@@ -962,8 +963,8 @@ static inline int MPIDI_NM_mpi_win_allocate_shared(MPI_Aint size,
     if (mpi_errno != MPI_SUCCESS)
         goto fn_fail;
 
-    sprintf(shm_key, "/mpi-%X-%X-%" PRIx64, MPIDI_Global.jobid, root_rank,
-            MPIDI_OFI_WIN(win).win_id);
+    snprintf(shm_key, MPIDI_OFI_MAX_SHMKEY_LEN, "/mpi-%X-%X-%" PRIx64, MPIDI_Global.jobid, root_rank,
```

Comment:
Now that we are using `snprintf`, can we check the return value to make sure we didn't run out of space?